src/e2e-test/features/bigquerymultitable/sink/GCSToBQMT.feature

```diff
@@ -0,0 +1,285 @@
+@BQMT_Sink
+Feature: BQMT-Sink - Verify BQMT sink plugin data transfer scenarios
```

Comment:
BQMT-Sink - Verify GCS to BQMT sink plugin data transfer scenarios

Reply:
Change done.